site/style.css

```diff
@@ -1,11 +1,137 @@
+body {
+  margin:0;
+  }
+
+h1 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 60px;
+  color: #FFF;
+  }
+
+h2 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 40px;
+  color: #020D38;
+  }
+
+h3 {
+  font-family: 'Lato', sans-serif;
+  font-size: 30px;
+  color: #020D38;
+  }
+
+h4 {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  color: #06259E;
+  }
+
 p {
-	color: red;
-}
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  }
+
+a {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  color: #FFF;
+  }
 
-.greenparagraph {
-	color: blue;
+tr, td {
+  padding: 10px;
 }
 
-p span {
-	color: #831FB5;
-}
\ No newline at end of file
+li {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  }
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  }
+
+li a:hover:not(.active) {
+  background-color: #172F5C;
+  }
+
+.active {
+  background-color: #E06B04;
+  }
+
+.top_nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #0D0740;
+  }
+
+.top_nav_item {
+  float: left;
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  }
+
+.name_area {
+  background-color: #06259E;
+  padding: 30px;
+  text-align: center;
+  }
+
+.social_media {
+  background-color: #020D38;
+  padding-left: 40px;
+  padding-top: 30px;
+  padding-bottom: 50px;
+  }
+
+.joe_pic {
+  float: left;
+  margin-right: 5px;
+  margin-bottom: 10px;
+  }
+
+.description {
```

Comment:
Now that we have talked more about margin (and padding), think about how you can combine these 4 rules into a single margin (shorthand) rule, like we talked about in the last class.